freeipa.spec.in

```diff
@@ -1271,6 +1271,7 @@ fi
 
 %post client
 if [ $1 -gt 1 ] ; then
+    %triggerin client -- sssd-common >= 2.10
```

Comment:
You can have a look at line https://github.com/freeipa/freeipa/blob/f9f96ac4a802e9b38d156fddbc98592ac0981726/freeipa.spec.in#L1325 for an example. The scriptlet must be able to handle upgrade or downgrade of sssd-common and has a section on its own, outside of the `%post client` section.